tests/unit/test_unit.py

```diff
@@ -30,18 +30,6 @@ def test_api_docs(client):
      resp = client.get(url_for('api_docs.api_docs'))
      assert resp.status_code == 200
 
-def test_api_private_asn_by_month(client):
-    resp = client.get(url_for('api_private.api_private_asn_by_month'))
-    assert resp.status_code == 200
-
-def test_api_private_counties_by_month(client):
-    resp = client.get(url_for('api_private.api_private_counties_by_month'))
-    assert resp.status_code == 200
-
-def test_api_private_runs_by_month(client):
-    resp = client.get(url_for('api_private.api_private_runs_by_month'))
-    assert resp.status_code == 200
-
```

Comment:
Why have these tests been deleted?

Reply:
Because they were failing non-deterministically (due to the use of the caching flask endpoint, which we should also BTW stop using) and are in any case unittests for private API methods that I don't think we should be unittesting anyways.

Reply:
Fair enough.